sycl/test-e2e/Compression/Inputs/single_kernel.cpp

```diff
@@ -0,0 +1,17 @@
+#include <sycl/detail/core.hpp>
+
+int main() {
+
+  sycl::queue q0;
+  int val = -1;
+  {
+    sycl::buffer<int, 1> buffer1(&val, sycl::range(1));
+
+    q0.submit([&](sycl::handler &cgh) {
```

Comment:
The description on this PR says that there is a threshold of 512 bytes, below which we don't compress.

Is this tiny kernel above or below that threshold?  And shouldn't we have a test for the other side of that threshold as well?

Reply:
> Is this tiny kernel above or below that threshold? 

It is above the threshold maybe because I use `-O0` to compile :P

>  And shouldn't we have a test for the other side of that threshold as well?

Yes, I have the following test to ensure that there's no compression when the size < threshold:
[clang/test/Driver/clang-offload-wrapper-zstd.c](https://github.com/intel/llvm/pull/15124/files#diff-a8f20e0fdb0b70c22204d8c7fb22aeb6d3d9b65258e5c6a8e9223db699e3cd95)